Source/Core/DolphinQt/Settings/AudioPane.cpp

```diff
@@ -24,19 +20,25 @@
 #include "Core/Config/MainSettings.h"
 #include "Core/Core.h"
 #include "Core/System.h"
+#include "DolphinQt/Config/ConfigControls/ConfigBool.h"
+#include "DolphinQt/Config/ConfigControls/ConfigChoice.h"
+#include "DolphinQt/Config/ConfigControls/ConfigRadio.h"
+#include "DolphinQt/Config/ConfigControls/ConfigSlider.h"
 
 #include "DolphinQt/Config/ConfigControls/ConfigBool.h"
 #include "DolphinQt/Config/SettingsWindow.h"
 #include "DolphinQt/Settings.h"
 
+constexpr int DSP_HLE_INDEX = 0;
```

Comment:
Can't you get the same result using `Config::Get(MAIN_DSP_HLE)` instead? When I first tested this PR, the `enabled` checks weren't working properly (e.g. Dolby Pro Logic II enabled when the combobox was blank).